Packs/Censys/Integrations/CensysV2/CensysV2.py

```diff
@@ -0,0 +1,213 @@
+        url_suffix = 'v2/hosts/search'
+        params = {
+            'q': query,
+            'per_page': page_size
+        }
+        res = self._http_request('GET', url_suffix, params=params)
+        return res
+
+    def censys_search_certs_request(self, data: Dict) -> Dict:
+        url_suffix = 'v1/search/certificates'
+        res = self._http_request('POST', url_suffix, json_data=data)
+        return res
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    client.censys_view_request('ipv4', '8.8.8.8')
+    return 'ok'
+
+
+def censys_view_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Returns host information for the specified IP address or structured certificate data for the specified SHA-256
+    """
+    index = args.get('index', '')
+    query = args.get('query', '')
+    res = client.censys_view_request(index, query)
+    if index == 'ipv4':
+        result = res.get('result', {})
+        content = {
+            'Name': result.get('autonomous_system', {}).get('name'),
+            'Bgp Prefix': result.get('autonomous_system', {}).get('bgp_prefix'),
+            'ASN': result.get('autonomous_system', {}).get('asn'),
+            'Service': [{
+                'Port': service.get('port'),
+                'Service Name': service.get('service_name')
+            } for service in result.get('services', [])],
+            'Last Updated': result.get('last_updated_at')
+        }
+
+        human_readable = tableToMarkdown(f'Information for IP {query}', content)
+        return CommandResults(
+            readable_output=human_readable,
+            outputs_prefix='Censys.View',
+            outputs_key_field='ip',
+            outputs=result,
+            raw_response=res
+        )
+    else:
+        metadata = res.get('metadata', {})
+        content = {
+            'SHA 256': res.get('fingerprint_sha256'),
+            'Tags': res.get('tags'),
+            'Source': metadata.get('source'),
+            'Added': metadata.get('added_at'),
+            'Updated': metadata.get('updated_at')
+        }
+        human_readable = tableToMarkdown('Information for certificate', content)
+        return CommandResults(
+            readable_output=human_readable,
+            outputs_prefix='Censys.View',
+            outputs_key_field='fingerprint_sha256',
+            outputs=res,
+            raw_response=res
+        )
+
+
+def censys_search_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Returns previews of hosts matching a specified search query or a list of certificates that match the given query.
+    """
+    index = args.get('index')
+    query = args.get('query', '')
+    page_size = arg_to_number(args.get('page_size', 50))  # type: ignore[assignment]
+    limit = arg_to_number(args.get('limit'))
+    contents = []
+
+    if index == 'ipv4':
+        if limit and limit < page_size:  # type: ignore
```

Comment:
now that you're ignoring line 103, you can remove the `type: ignore` here and in the line beliw